topics/journalpost.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: kafka.nais.io/v1
+kind: Topic
+metadata:
+  name: journalpost
+  namespace: bidrag
+  labels:
+    team: bidrag
+spec:
+  pool: {{kafka-pool}}
+  config:
+    cleanupPolicy: compact
```

Comment:
Alternativet her er vel "delete"? Og det vil vi vel ikke ha.

Reply:
stemmer

Reply:
Nettopp, for det er når man kjører cleanup manuelt dette har effekt?

Reply:
dette er utenfor min komfortsone 😞 ... jeg vet ikke... jeg leste "et sted" at compact betyr at den siste er garantert lagret...